composer.json

```diff
@@ -39,7 +39,9 @@
     "phpmd/phpmd": "2.13.0",
     "phpunit/phpunit": "10.1.2 || 9.6.7 || 8.5.31 || 7.5.20 || 6.5.14 || 5.7.27 || 4.8.36",
     "sebastian/phpcpd": "6.0.3 || 5.0.2 || 4.1.0 || 3.0.1 || 2.0.4",
-    "squizlabs/php_codesniffer": "3.7.2 || 2.9.2"
+    "squizlabs/php_codesniffer": "3.7.2 || 2.9.2",
+    "phpstan/phpstan": "^1.10",
+    "phpstan/phpstan-deprecation-rules": "^1.1"
```

Comment:
The suggestion to use phpstan is good but, wouldn't this break older php versions?
Maybe we can add it to the GitHub actions like tcpdf lib or via Makefile.